server/plugin/plg_editor_onlyoffice/index.go

```diff
@@ -184,7 +185,7 @@ func IframeContentHandler(ctx App, res http.ResponseWriter, req *http.Request) {
 		return
 	}
 	key = HashStream(f, 20)
-	key = Hash(key + userId + path, 20)
+	key = Hash(key + path, 20)
```

Comment:
I'm not sure why did you remove the userId there? It's a safety mechanism

Reply:
I wrote why I had to do that in the second paragraph of my first comment